src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+import './assets/images/reload.png';
+import './assets/images/enter.png';
+import './assets/images/more.png';
+
+const tasks = [
+  {
+    desc: 'Wash the dishes',
+    completed: false,
+    index: 1,
+  },
+  {
+    desc: 'Complete The Microverse',
+    completed: false,
+    index: 2,
+  },
+  {
+    desc: 'Go to a travel',
+    completed: false,
+    index: 3,
+  },
+  {
+    desc: 'Find a good job',
+    completed: false,
+    index: 4,
+  },
```

Comment:
- [ ] The tasks are not being displayed on the page. Kindly ensure they are displayed as we have in the video.